mne/cov.py

```diff
@@ -1713,6 +1718,8 @@ def _estimate_rank_meeg_signals(data, info, scalings, tol=1e-4,
         thresholded to determine the rank are also returned.
     """
     picks_list = _picks_by_type(info)
+    if scalings is None:
+        scalings = DEFAULTS['scalings']
```

Comment:
Seems OK calling from <code>raw.estimate_rank(scalings='norm')</code> and then <code>raw.estimate_rank(scalings=None)</code>, but I'll wait for @dengemann to reply regarding potential reasons he used that syntax, assuming he wrote that bit of code.